index.js

```diff
@@ -0,0 +1,10 @@
+var express = require( "express" ),
+	app, server, config;
+config = {
+	port: 3000
```

Comment:
Is there a way we could not hardcode the port? Based on an environment variable and a default value, for example?